File: core/src/main/java/com/fahamutech/kcore/admin/database/connector/ArticleDataSource.java
package com.fahamutech.kcore.admin.database.connector;

import androidx.swiperefreshlayout.widget.SwipeRefreshLayout;
import androidx.recyclerview.widget.RecyclerView;


public interface ArticleDataSource {
    void getAllById(String categoryId, RecyclerView recyclerView,
                    SwipeRefreshLayout swipeRefreshLayout);

}

File: core/src/main/java/com/fahamutech/kcore/admin/database/connector/ArticleDataSource.kt
package com.fahamutech.kcore.admin.database.connector

import androidx.swiperefreshlayout.widget.SwipeRefreshLayout
import androidx.recyclerview.widget.RecyclerView

interface ArticleDataSource {
    fun getAllById(
        categoryId: String?, recyclerView: RecyclerView?,
        swipeRefreshLayout: SwipeRefreshLayout?
    )
}